terratest/src/test/backingservices/HelmChartParser.go

```diff
@@ -0,0 +1,54 @@
+package backingservices
```

Comment:
This looks like duplicate code from other package. If it needs to be reused, can this be moved to a common package if possible.